AElf.Kernel.Core.Tests/KernelCoreTestAElfModule.cs

```diff
@@ -19,15 +16,33 @@ namespace AElf.Kernel
         typeof(TestBaseKernelAElfModule))]
     public class KernelCoreTestAElfModule : AElfModule
     {
+        delegate void MockGenerateTransactions(Address @from, long preBlockHeight, Hash previousBlockHash,
+            ref List<Transaction> generatedTransactions);
         public override void ConfigureServices(ServiceConfigurationContext context)
         {
             var services = context.Services;
             services.AddTransient<BlockValidationProvider>();
+            
+            services.AddTransient(provider =>
+            {
+                var transactionList = new List<Transaction>
+                {
+                    new Transaction() {From = Address.Zero, To = Address.Generate(), MethodName = "InValue"},
+                    new Transaction() {From = Address.Zero, To = Address.Generate(), MethodName = "OutValue"},
+                };
+                var consensusTransactionGenerator = new Mock<ISystemTransactionGenerator>();
+                consensusTransactionGenerator.Setup(m => m.GenerateTransactions(@It.IsAny<Address>(), It.IsAny<long>(),
```

Comment:
`@It`

Reply:
Removed all '@' symbols in test, sorry for did't understand the symbol meaning used with parameter.